### PR TITLE
migrate rethbadger liq script

### PR DESCRIPTION
**Pull request by jalbrekt85**
handles #1145
```
brownie run issue/1145/migrate_rethbadger
```

### CODE REVIEW

File: scripts/issue/1145/migrate_rethbadger.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+    vault.balancer.deposit_and_stake(
+        [badger, reth],
+        [badger_to_deposit, reth_to_deposit],
+        pool=bpt_badgerreth_new,
+        stake=False,
+        pool_type="Weighted",
+    )
+
+    # aura deposit fails; no aura pool id for new pool
```
Comment (gosuto-inzasheru) [inline scripts/issue/1145/migrate_rethbadger.py:43]: aura says they have a tx queued to add the new pools soon

**Review by gosuto-inzasheru — COMMENTED** [review]
greatapesafe ftw

am in contact with aura in regards to the new pool id, will update here

**Review by gosuto-inzasheru — APPROVED** [review]
Screenshot 2023-02-22 at 11 37 55

https://etherscan.io/tx/0x87dac6c9c7ff95d0a12268a3566e944c26dfb564986227e33b5cfa9d458e5090#eventlog

script runs successfully now except for the preview.

**Review by gosuto-inzasheru — APPROVED** [review]
runs successfully, including preview now. had to reload some contacts from explorer locally.

L G T M

**Review by petrovska-petro — APPROVED** [review]
- confirmed new balancer pool addrs ✅ 
- deposits in the right pid after aura had added new pools in the booster ✅ 
- not dusty by lev depositAll and stake ✅

**Pull request merged**